tests/ImageSharp.Tests/TestImages.cs

```diff
@@ -160,6 +160,8 @@ public static class Issues
                 public const string OrderedInterleavedProgressive723A = "Jpg/issues/Issue723-Ordered-Interleaved-Progressive-A.jpg";
                 public const string OrderedInterleavedProgressive723B = "Jpg/issues/Issue723-Ordered-Interleaved-Progressive-B.jpg";
                 public const string OrderedInterleavedProgressive723C = "Jpg/issues/Issue723-Ordered-Interleaved-Progressive-C.jpg";
+                public const string ExifGetString750Transform = "Jpg/issues/issue750-exif-tranform.jpg";
```

Comment:
Small typo here. Thanks for adding these images.

Reply:
No worries, I'll update the reference file name later on, will do for now.